### PR TITLE
Update AUTHORS/.mailmap

### PR DESCRIPTION
**Pull request by asmeurer**
Also prevent azure-pipelines[bot] from being added to AUTHORS.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -744,7 +744,7 @@ JMSS-Unknown <31131631+JMSS-Unknown@users.noreply.github.com>
 Arshdeep Singh <singh.arshdeep1999@gmail.com>
 cym1 <16437732+cym1@users.noreply.github.com>
 Stewart Wadsworth <stewart.wadsworth@gmail.com>
-Jared Lumpe <mjlumpe@gmail.com>
+Michael Jared Lumpe <mjlumpe@gmail.com>
```
Comment (asmeurer) [inline AUTHORS]: @jlumpe do you prefer "Michael Jared Lumpe" or "Jared Lumpe" in the AUTHORS file?

File: .mailmap [hunk 2]
```diff
@@ -394,3 +394,5 @@ Gagandeep Singh <singh.23@iitj.ac.in> czgdp1807 <gdp.1807@gmail.com>
 Gagandeep Singh <singh.23@iitj.ac.in> czgdp1807 <36567889+czgdp1807@users.noreply.github.com>
 S.Y. Lee <sylee957@gmail.com> sylee957 <sylee957@gmail.com>
 Yatna Verma <yatnavermaa@gmail.com> yatna <yatnavermaa@gmail.com>
+czgdp1807 <czgdp1807@gmail.com> MathLover <36567889+czgdp1807@users.noreply.github.com>
```
Comment (asmeurer) [inline .mailmap]: @czgdp1807 is there a name you would prefer to have listed in the AUTHORS file?

**Comment by czgdp1807** [comment]
I am Gagandeep Singh with my official institute email id of singh.23@iitj.ac.in
I use czgdp1807@gmail.com and MathLover for GitHub only. It would be great if you remove `czgdp1807 <czgdp.1807@gmail.com> MathLover` and keep `Gagandeep Singh<singh.23@iitj.ac.in> czgdp1807`.
Apologies for the inconvenience caused. :-)

**Comment by asmeurer** [comment]
Ah, I didn't notice that you were already in there. I have fixed it.

File: AUTHORS (hunk 1 above)
Comment (jlumpe) [reply AUTHORS]: I think I'm trying to keep everything as just "Jared Lumpe" everywhere. Thanks!

Comment (asmeurer) [reply AUTHORS]: Fixed.

**Pull request merged**